cmd/algod/main.go

```diff
@@ -157,18 +158,24 @@ func run() int {
 		return 1
 	}
 	defer fileLock.Unlock()
-
+	
 	cfg, err := config.LoadConfigFromDisk(absolutePath)
 	if err != nil && !os.IsNotExist(err) {
 		// log is not setup yet, this will log to stderr
 		log.Fatalf("Cannot load config: %v", err)
 	}
-
+	
 	err = config.LoadConfigurableConsensusProtocols(absolutePath)
 	if err != nil {
 		// log is not setup yet, this will log to stderr
 		log.Fatalf("Unable to load optional consensus protocols file: %v", err)
 	}
+	
+	if logToStdout != nil && *logToStdout {
+		cfg.LogSizeLimit = 0
+		log.UsePrettyOutput(os.Stdout)
```

Comment:
Consider leaving out the pretty output or making it a separate flag. This feature was added for docker and I don't know if it would treat each newline separated line as a separate entry.